minilzo/PSPBUILD

```diff
@@ -0,0 +1,36 @@
+pkgname=minilzo
+pkgver=2.09
+pkgrel=1
+pkgdesc="mini subset of the LZO real-time data compression library"
```

Comment:
I did a quick check and it seems we start all the pkgdesc with a capital letter:
```suggestion
pkgdesc="Mini subset of the LZO real-time data compression library"
```